src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -308,6 +309,31 @@ class RunSetupPyResult:
     output: Digest  # The state of the chroot after running setup.py.
 
 
+class PythonDistributionSubsystem(Subsystem):
+    """Options for packaging wheels/sdists from a `python_distribution` target."""
+
+    options_scope = "python-distribution"
+
+    @classmethod
+    def register_options(cls, register):
+        super().register_options(register)
+        register(
+            "--exact-first-party-dependency-requirements",
```

Comment:
Very open to a better name. This is...chatty.

Reply:
Perhaps it should be an enum? In the future, a user might want to have no constraints at all.